docs/src/pages/components/view/demo.tsx

```diff
@@ -1,286 +1,82 @@
-          <input
-            type="text"
-            placeholder="Set maxWidth"
-            value={maxWidth}
-            onChange={(event: any) => {
-              setMaxWidth(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="minHeight">
-          <input
-            type="text"
-            placeholder="Set minHeight"
-            value={minHeight}
-            onChange={(event: any) => {
-              setMinHeight(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="minWidth">
-          <input
-            type="text"
-            placeholder="Set minWidth"
-            value={minWidth}
-            onChange={(event: any) => {
-              setMinWidth(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="role">
-          <input
-            type="text"
-            placeholder="Set role"
-            value={role}
-            onChange={(event: any) => {
-              setAriaRole(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="opacity">
-          <input
-            type="text"
-            placeholder="Set opacity"
-            value={opacity}
-            onChange={(event: any) => {
-              setOpacity(event.target.value);
-            }}
-          />
-        </FieldSet>
+    `<View` +
+    (props.ariaLabel ? `\n  ariaLabel="${props.ariaLabel}"` : '') +
+    `\n  as="${props.as}"` +
+    (props.backgroundColor
+      ? `\n  backgroundColor="${props.backgroundColor}"`
+      : '') +
+    (props.borderRadius ? `\n  borderRadius="${props.borderRadius}"` : '') +
+    (props.border ? `\n  border="${props.border}"` : '') +
+    (props.boxShadow ? `\n  boxShadow="${props.boxShadow}"` : '') +
+    (props.color ? `\n  color="${props.color}"` : '') +
+    (props.height ? `\n  height="${props.height}"` : '') +
+    (props.opacity ? `\n  opacity="${props.opacity}"` : '') +
+    (props.padding ? `\n  padding="${props.padding}"` : '') +
+    (props.width ? `\n  width="${props.width}"` : '') +
+    `\n  onClick={() => alert('🏔 What a beautiful <View>! 🔭')}
+  >
+  {"I\'m a <${props.as}>! 🤩"}
+</View>`
+  );
+};
 
-        <FieldSet id="padding">
-          <input
-            type="text"
-            placeholder="Set padding"
-            value={padding}
-            onChange={(event: any) => {
-              setPadding(event.target.value);
-            }}
-          />
-        </FieldSet>
+export const ViewDemo = () => {
+  const props = useViewProps({
+    as: 'div',
+    ariaLabel: 'View example',
+    width: '20rem',
+    height: '4rem',
+    color: 'var(--amplify-colors-blue-60)',
+    backgroundColor: 'var(--amplify-colors-white)',
+    boxShadow: '3px 3px 5px 6px var(--amplify-colors-neutral-60)',
+    padding: '1rem',
+    border: '1px solid var(--amplify-colors-black)',
+    borderRadius: '6px',
+    opacity: '100%',
```

Comment:
I don't see the opacity prop as an option in the demo?

Reply:
Good point, I forgot to remove it here. Will update.